website/docs/cloud-docs/api-docs/workspaces.mdx

```diff
@@ -861,6 +861,7 @@ This endpoint supports pagination [with standard URL query parameters](/terrafor
 | `search[tags]`          | **Optional.** If specified, restricts results to workspaces with that tag. If multiple comma separated values are specified, results matching all of the tags are returned.                                                                                                                                                                                                                                          |
 | `search[exclude-tags]`  | **Optional.** If specified, results exclude workspaces with that tag. If multiple comma separated values are specified, workspaces with tags matching any of the tags are excluded.                                                                                                                                                                                                                                  |
 | `search[wildcard-name]` | **Optional.** If specified, restricts results to workspaces with partial matching, using `*` on prefix, suffix, or both. For example, `search[wildcard-name]=*-prod` returns all workspaces ending in `-prod`, `search[wildcard-name]=prod-*` returns all workspaces beginning with `prod-`, and `search[wildcard-name]=*-prod-*` returns all workspaces with substring `-prod-` regardless of prefix and/or suffix. |
+| `sort`                  | **Optional.** Allows sorting the returned workspaces. Valid values are `"name"`, `"current-run.created-at"` (which sorts by the time of the current run), and `"latest-change-at"` (which sorts by creation time of the current state version, or of the workspace itself if no state versions exist). Prepending a hyphen to the sort parameter will reverse the order (e.g. `"-name"`). The default sort order is arbitrary but stable, and cannot be explicitly specified.                                                                                                                                                     |
```

Comment:
Adding some small suggestions for style: 
```suggestion
| `sort`                  | **Optional.** Allows sorting the organization's workspaces by a provided value. You can sort by `"name"`, `"current-run.created-at"` (the time of the current run), and `"latest-change-at"` (the creation time of the latest state version or the workspace itself if no state version exists). Prepending a hyphen to the sort parameter reverses the order. For example, `"-name"` sorts by name in reverse alphabetical order. If omitted, the default sort order is arbitrary but stable.                                                                                                                                                     |
```
I think we can remove "and cannot be explicitly specified" from the last sentence, because it seems like you can explicitly specify the sort order using this query parameter.

Reply:
Oooh, thank you for the background context here! It's always cool to know how things work behind the scenes! 🌟